src-backbone/app/css/modules/builder.less

```diff
@@ -71,14 +71,24 @@ header#builder-meta-data{
         }
     }
 
-    a#download-btn{
+    a.builder-header-btn{
         margin-top: -20px;
         position: absolute;
-        top:50%; right: @container-side-padding;
+        top:50%;
 
         font-weight: bold;
         height: 40px;
         line-height: 40px;
         padding: 0 20px;
     }
+
+    a#download-btn{
+        &:extend(.builder-header-btn);
+        right: @container-side-padding;
+    }
+
+    a#save-btn{
+        &:extend(.builder-header-btn);
+        right: @container-side-padding * 3;
```

Comment:
this is assuming the download button is a fixed width

wrap the buttons with a parent container and absolute position the parent